Content.Shared/Disease/DiseasePrototype.cs

```diff
@@ -0,0 +1,62 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+/// <summary>
+/// Diseases encompass everything from viruses to cancers to heart disease.
+/// It's not just a virology thing.
+/// </summary>
+namespace Content.Shared.Disease
+{
+    [Prototype("disease")]
+    [DataDefinition]
+    public sealed class DiseasePrototype : IPrototype, IInheritingPrototype
+    {
+        [ViewVariables]
+        [DataField("id", required: true)]
+        public string ID { get; } = default!;
+        /// <summary>
+        /// This one is more important than you might think since
+        /// due to the way prototypes work
+        /// This is how a disease identifies another instance of itself
+        /// </summary>
+        [DataField("name")]
+        public string Name { get; } = string.Empty;
+
+        [DataField("parent", customTypeSerializer: typeof(PrototypeIdSerializer<DiseasePrototype>))]
+        public string? Parent { get; private set; }
+
+        [NeverPushInheritance]
+        [DataField("abstract")]
+        public bool Abstract { get; private set; }
+        public float Accumulator = 0f;
```

Comment:
..why does the prototype need an accumulator, that seems really sus

Reply:
That's used to control the tick rate of all the effects and cures (1 second by default)

Reply:
i guess thats ok